### PR TITLE
[M1429-2] SQL Models

### PR DESCRIPTION
**Pull request by gridcell**

### CODE REVIEW

**Comment by gridcell** [comment]
[improve aggregated endpoint performance](https://trello.com/c/VmhKeFJN/1429-improve-aggregated-endpoint-performance)

File: src/api/scripts/test_sql_models.py [hunk 1]
```diff
@@ -0,0 +1,300 @@
+from api.models import (
```
Comment (saanobhaai) [inline src/api/scripts/test_sql_models.py:1]: These are great... are you thinking we'll remove the MV versions down the road? If so we should create a ticket for that, so we don't forget. Also we already merged in the SQS code which we're not (yet) using... all fine, just don't want a lot of code hanging around that we're not using.

File: src/sqltables/query.py [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+                    params = []
+
+                alias = self.join(
+                    SQLTable(self.get_meta().db_table, None, params, self.model_sql)
+                )
+            else:
+                alias = self.join(BaseTable(self.get_meta().db_table, None))
+        return alias
+
+    def setup_joins(  # noqa: C901
```
[the remark this answers is not in the document]
Comment (gridcell) [reply src/sqltables/query.py:45]: telling flake8 to ignore the complexity of the method

File: src/api/scripts/test_sql_models.py (hunk 1 above)
Comment (gridcell) [reply src/api/scripts/test_sql_models.py:1]: we can chat about this further

**Pull request merged**